libs/sdk-ui-dashboard/src/_staging/dashboard/dashboardLayout.ts

```diff
@@ -261,13 +267,22 @@ export function dashboardLayoutRemoveIdentity<T extends IDashboardWidget>(
                     return item;
                 }
 
+                /**
+                 * We want to keep localIdentifier which is the only widget identity stored on backend.
+                 * If it is nonexistent, we create a new one.
+                 */
+                const localIdentifierObj = isWidgetIdentifiersEnabled
+                    ? { localIdentifier: item.widget?.localIdentifier ?? uuidv4() }
+                    : { localIdentifier: undefined };
```

Comment:
Should this be rather empty, in case it's turned off?

Reply:
true, will fix that